### ISSUE TITLE
[Bug]: TestStatsLogWriter UT failed

### ISSUE THREAD

**Issue by zhangxu19830126** [matrixorigin/matrixone#8508]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

2023-03-17T06:42:26.1241453Z === RUN   TestStatsLogWriter
2023-03-17T06:42:26.1242195Z 2023/03/17 06:42:22.048030 +0000 INFO cn-service mometric/stats_log_writer.go:86 MockServiceStats stats  {"uuid": "test", "reads": 2, "hits": 1}
2023-03-17T06:42:26.1243080Z 2023/03/17 06:42:24.048035 +0000 INFO cn-service mometric/stats_log_writer.go:86 MockServiceStats stats  {"uuid": "test", "reads": 0, "hits": 0}
2023-03-17T06:42:26.1243805Z StatsLogWriter has stopped gracefully.
2023-03-17T06:42:26.1244310Z     stats_log_writer_test.go:114: 
2023-03-17T06:42:26.1245267Z         	Error Trace:	/runner/_work/matrixone/matrixone/pkg/util/metric/mometric/stats_log_writer_test.go:114
2023-03-17T06:42:26.1245857Z         	Error:      	Not equal: 
2023-03-17T06:42:26.1246393Z         	            	expected: 3
2023-03-17T06:42:26.1246925Z         	            	actual  : 2
2023-03-17T06:42:26.1247419Z         	Test:       	TestStatsLogWriter
2023-03-17T06:42:26.1247941Z --- FAIL: TestStatsLogWriter (6.00s)
2023-03-17T06:42:26.1248290Z FAIL
2023-03-17T06:42:26.1248736Z FAIL	github.com/matrixorigin/matrixone/pkg/util/metric/mometric	41.446s

**Comment by zhangxu19830126** [comment]
@arjunsk i think it's yours

**Comment by arjunsk** [comment]
@zhangxu19830126 Could you kindly close this issue? I think it is reopening on its own.

**Comment by arjunsk** [comment]
Hi @ @zhangxu19830126, this issue was resolved in #8510. Kindly close this issue.
